apollo-api/src/main/java/com/apollographql/apollo/api/Operation.java

```diff
@@ -49,7 +52,7 @@
   /**
    * Parses provided GraphQL operation raw response
    *
-   * @param source for operation raw response to parse
+   * @param source             for operation raw response to parse
```

Comment:
typo ?

Reply:
I think the Java code style forces param definitions to be on the same column. I noticed this in earlier PRs too. Would be good to align.

Reply:
Yeah, it's formatting JavaDocs, will check how to disable this rule.

Reply:
Should we switch to dokka anyways ?

Reply:
I guess when we migrate these interfaces to Kotlin